### PR TITLE
Daniela-Parra-Intro-Python-II

### PR DESCRIPTION
**Pull request by danielaparra**

### CODE REVIEW

File: src/adv.py [hunk 1]
```diff
@@ -56,7 +56,7 @@
     print(player.roomCurrentlyIn.name)
     print(player.roomCurrentlyIn.description)
     print(player.roomCurrentlyIn.itemsInventory)
-    print("Things you can do: enter a cardinal direction (n, s, w, e), quit (q) or action (drop item)")
+    print("Things you can do: enter a cardinal direction (n, s, w, e), quit (q) or action (get/take item)")
```
Comment (LuisanSuarez) [inline src/adv.py]: nice addition. great ux. 

are you handling typos and alternative spellings?

**Comment by LuisanSuarez** [comment]
looking good so far, nice job keeping the code clean. let me know if you need any help!

File: src/adv.py [hunk 2]
```diff
@@ -54,17 +54,18 @@
...
     print(player.roomCurrentlyIn.itemsInventory())
-    print("Things you can do: move (n, s, w, e), quit (q) or action (take/drop item)")
+    print("Things you can do: move (n, s, w, e), quit (q) or action (take/drop item)\n")
     userInput = input(">> What would you like to do? ")
     
     inputWords = userInput.split(' ')
 
     if len(inputWords) == 1:
-        if userInput == "n":
+        if userInput == "n" or userInput == "north":
```
Comment (LuisanSuarez) [inline src/adv.py:88]: nice job making it robust. there's a nice Python feature where you can get the same result like this

if userInput in ['n', 'north']:
        #execute

also, try accounting for upper and lowercase

File: src/adv.py [hunk 3]
```diff
@@ -87,14 +87,16 @@
...
+        elif userInput == "w" or userInput == "west":
             player.move_w()
-        elif userInput == "e":
+        elif userInput == "e" or userInput == "east":
             player.move_e()
-        elif userInput == "q":
+        elif userInput == "q" or userInput == "quit":
             quit()
+        elif userInput == "i" or userInput == "inventory":
+            print(player.inventory())
```
Comment (LuisanSuarez) [inline src/adv.py:99]: I realize I'm commenting this as it's coming in, but: is this inventory() function named like so in your player.py file? I may have missed it, but I've only seen an i() function that seems to fill this purpose.

Comment (danielaparra) [reply src/adv.py:99]: Yeah I made an i and inventory that do the same thing I realized.

**Pull request merged**